src/util.c

```diff
@@ -173,6 +173,21 @@ int stringmatch(const char *pattern, const char *string, int nocase) {
     return stringmatchlen(pattern,strlen(pattern),string,strlen(string),nocase);
 }
 
+/* Return true if the string contains any of the characters in the
+ * null terminated string matchers. */
+int stringContains(const char *string, int stringlen, const char *matchers) {
```

Comment:
can't you use the existing `mempbrk` or maybe even the standard `strpbrk` in this case?

Reply:
What a nice function I couldn't find quickly googling.